src/algebra/big_operators/multiset.lean

```diff
@@ -151,6 +151,15 @@ end
 
 end comm_monoid
 
+lemma prod_dvd_prod' [comm_monoid β] {S : multiset α} (g1 g2 : α → β) (h : ∀ a ∈ S, g1 a ∣ g2 a) :
```

Comment:
```suggestion
lemma prod_dvd_prod_of_dvd [comm_monoid β] {S : multiset α} (g1 g2 : α → β) (h : ∀ a ∈ S, g1 a ∣ g2 a) :
```

Reply:
should this not just have the same name as the `finset` lemma? I don't mind which one gets changed, though, they both are reasonable

Reply:
I agree. It seems `multiset.prod_dvd_prod` it's not used a lot, so it shouldn't take too much time to rename it.

Reply:
Ok, l do that.  For simplicity would it be better to merge this PR as-is and then do the renaming in a separate PR?

Reply:
You can do it here, it's a small PR.

Reply:
Ok

Reply:
Since this got merged before I did the above renaming I've done it in #11734.